core/src/main/java/io/aiven/klaw/service/AclControllerService.java

```diff
@@ -280,7 +280,20 @@ public List<AclRequestsResponseModel> getAclRequests(
             tenantId);
 
     aclReqs = filterAclRequestsByTenantAndOrder(userName, aclReqs, order);
-    aclReqs = getAclRequestsPaged(aclReqs, pageNo, currentPage, tenantId);
+    aclReqs =
+        Pager.getItemsList(
+            pageNo,
+            currentPage,
+            10,
```

Comment:
may be we can create a constant in helpers for this recs per page?

Reply:
In fact there is one constant for 20 per page, 
the issue is that there are different values in different classes so, definitely  one constant doesn't cover all cases ...